spec/intl.html

```diff
@@ -927,6 +927,38 @@ <h1>Temporal.Calendar.prototype.dateUntil ( _one_, _two_, _options_ )</h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sup-temporal.calendar.prototype.era">
+          <h1>Temporal.Calendar.prototype.era ( _dateOrDateTime_ )</h1>
+          <p>
+            The `era` method takes one argument _dateOrDateTime_.
```

Comment:
```suggestion
            The `era` method takes one argument, _dateOrDateTime_.
```
What kind of argument?